thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -256,37 +258,70 @@ export default Ember.Controller.extend({
   },
 
   /**
+   * Replay Flow Step 1 - Clones an alert function in preparation for replay.
+   * @method callCloneAlert
+   * @param {Number} functionId - the newly created function's id
+   * @return {Ember.RSVP.Promise}
+   */
+  callCloneAlert(functionId) {
+    const url = `/onboard/function/${functionId}/clone/cloned`;
+    return fetch(url, { method: 'post' }).then(checkStatus);
+  },
+
+  /**
+   * Replay Flow Step 2 - Replay cloned function
+   * @method callReplayStart
+   * @param {Number} clonedId - id of the cloned function
+   * @param {Object} startTime - replay start time stamp
+   * @param {Object} endTime - replay end time stamp
+   * @return {Ember.RSVP.Promise}
+   */
+  callReplayStart(clonedId, startTime, endTime) {
+    const url = `/detection-job/${clonedId}/replay?start=${startTime}&end=${endTime}&speedup=true`;
```

Comment:
Hi Steve, here we have a different configuration: hourly and daily metrics, speedup should be "false".